relay-base-schema/src/organization.rs

```diff
@@ -0,0 +1,27 @@
+//! Contains [`OrganizationId`] which is the ID of a Sentry organization and is currently a
+//! a wrapper over `u64`.
+
+use serde::{Deserialize, Serialize};
+
+/// The unique identifier of a Sentry organization
+#[derive(Copy, Clone, Debug, PartialEq, Eq, Ord, PartialOrd, Hash, Serialize, Deserialize)]
+pub struct OrganizationId(u64);
```

Comment:
Can add a unit test to make sure serialization is correct using `serde_json`.

Reply:
I will add one. Just for curiosity, do you know if it's currently being serialized to JSON somewhere? I didn't see anything

Reply:
In the `ProjectInfo` struct, that is de- and serialized from JSON.